infra/charts/turing/templates/envs-secret.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if not .Values.mlp.environmentConfigSecret.name }}
```

Comment:
This secret itself and Turing's `environmentConfigs` data only matter if the external secret `.Values.mlp.environmentConfigSecret.name` is not set, correct?

Reply:
Yes, that's correct.